internal/config/server_configuration.go

```diff
@@ -1,11 +1,16 @@
 package config
 
 type Server struct {
-	Port int `toml:"port"`
+	GSPort int `toml:"groundstation_port"`
+	WAPort int `toml:"webapi_port"`
 }
 
 type Database struct {
-	Url string `toml:"url"`
+	InMemory    bool   `toml:"inmemory"`
+	Postgres    bool   `toml:"postgres"`
+	PostgresUrl string `toml:"url"`
+	// TODO: this is currently unused - need to figure out how to pass toml config to tests
+	TestUrl string `toml:"test_url"`
```

Comment:
Tests shouldn't load from toml IMO.

Reply:
I'm happy for tests to read from toml if it's the most efficient way to set up the state for the testable code, but there is always the option of mocking whatever state instead. I think we should go for the latter option too.

Reply:
Yeah, I can go back and remove this because I don't think tests with TOML is doable due to them all being run in temp directories containing just the binary being tested

Reply:
Removed in 1229c0e